applications/CoSimulationApplication/tests/fsi_sdof/ProjectParametersCFD.json

```diff
@@ -180,7 +180,7 @@
             }
 
         }],
-        "auxiliar_process_list"            : [{
```

Comment:
to me this kind of changes is a blocker. IT would cause cases that were prevously running to fail.

you can (and you did) fix the tests, but this also breaks existing examples that users have on their own computer